pkgs/applications/misc/opentx/default.nix

```diff
@@ -0,0 +1,55 @@
+{ stdenv, fetchFromGitHub
+, cmake, gcc-arm-embedded, python
+, qt5, SDL, gmock
+, dfu-util, avrdude
+}:
+
+let
+
+  version = "2.2.0";
+
+in stdenv.mkDerivation {
+
+  name = "opentx-${version}";
+
+  src = fetchFromGitHub {
+    owner = "opentx";
+    repo = "opentx";
+    rev = version;
+    sha256 = "1warjvb3y6x1a6xb5i1d4m6ckj9ixrl7mv0c2ndfx9pxxg9v0zv5";
+  };
+
+  enableParallelBuilding = true;
+
+  buildInputs = [
+    cmake gcc-arm-embedded
+    python python.pkgs.pyqt4
+    qt5.full SDL gmock
```

Comment:
cmake should be in `nativeBuildInputs`

Reply:
Something in the cmakelist was checking for it, so I just added it and the warning went away. Any idea why I get this qt lib file-not-found build error on the newer master branch?

Reply:
Does cmake add a pyqt4 in its build or is this information thrown away?

Reply:
It's needed:

   -- Build files have been written to: /tmp/nix-build-opentx-2.2.0.drv-2/opentx-2.2.0-src/build
    building
    build flags: -j8 -l8 SHELL=/nix/store/h404wfcz8rzzlq8vr4z7plcijwzfci72-bash-4.4-p12/bin/bash
    Scanning dependencies of target taranis_xbm
    [  0%] Generating sticks.lbm
    Traceback (most recent call last):
      File "/tmp/nix-build-opentx-2.2.0.drv-2/opentx-2.2.0-src/radio/util/img2lbm.py", line 9, in <module>
        from PyQt4 import Qt, QtGui
    ImportError: No module named PyQt4
    make[2]: *** [radio/src/bitmaps/212x64/CMakeFiles/taranis_xbm.dir/build.make:61: radio/src/bitmaps/212x64/sticks.lbm] Error 1
    make[1]: *** [CMakeFiles/Makefile2:520: radio/src/bitmaps/212x64/CMakeFiles/taranis_xbm.dir/all] Error 2
    make: *** [Makefile:152: all] Error 2
    builder for ‘/nix/store/fafad88azc5mh0xdd9by1sflglws6hdn-opentx-2.2.0.drv’ failed with exit code 2
    error: build of ‘/nix/store/fafad88azc5mh0xdd9by1sflglws6hdn-opentx-2.2.0.drv’ failed

I still need help with the missing Qt5 files error.